components/ecu/ecu_firmware/Core/Src/can.c

```diff
@@ -4,21 +4,31 @@
  * 
  * @date 2021/01/30
  * @author Blake Shular (blake-shular)
+
+ * @param currentVal the current flowing through the ECU
+ * @param doc_coc Discharge and Charge overcurrent status
  */
 
 #include "can.h"
+#include "main.h"
+
+#define CAN_ID 0x450
+#define MESSAGE_LENGTH 3
+
 
 CAN_TxHeaderTypeDef current_message_header = {
-    0x624, 0, CAN_ID_STD, CAN_RTR_DATA, 6, DISABLE
+    CAN_ID, 0, CAN_ID_STD, CAN_RTR_DATA, MESSAGE_LENGTH, DISABLE
 }; 
 
-void CAN_send_current(int currentVal) {
+void CAN_send_current(int currentVal, int doc_coc) {
```

Comment:
Your function description comment (with the `@param` lines) needs to go right above this line